repos/dir.props

```diff
@@ -18,6 +31,16 @@
     <VersionFileLocation Condition="'$(VersionFileLocation)' == ''">$(VersionFileDirectory)$(RepositoryOrganization)/$(RepositoryName)/$(RepositoryBranch)/$(VersionFilename)</VersionFileLocation>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <OrchestratedManifestBuildName Condition="'$(OrchestratedManifestBuildName)' == ''">$(RepositoryName)</OrchestratedManifestBuildName>
+  </PropertyGroup>
+
+  <!-- for RTM only, default to stabilized builds because that's what matches the official builds -->
+  <!-- once https://github.com/dotnet/source-build/issues/550 is fixed we will be able to determine this automatically -->
+  <PropertyGroup>
+    <UseStableVersions Condition="'$(UseStableVersions)' == ''">true</UseStableVersions>
```

Comment:
Looks like you didn't get the `DropSuffix` change in CLI but did get this change - I think we want to be consistent in stable vs. unstable.  I think master makes sense to be unstable by default.

Reply:
I can update the PR to get the DropSuffix change in. 

>  I think master makes sense to be unstable by default.

So that would mean setting UseStableVersions to false by default and not getting the DropSuffix change in cli.proj right?

Reply:
Yes, that's right - @dagood does master unstable make sense to you?

Reply:
It sounds fine, I think the important thing is matching ProdCon though, or even being more stable than ProdCon to make sure we won't be broken when ProdCon stabilizes.

Reply:
So I updated UseStableVersions to false according to this.

Reply:
Yeah I think it makes sense to produce unstable builds especially as we start to take additional changes.